oak_functions/Dockerfile

```diff
@@ -1,17 +1,15 @@
 FROM gcr.io/distroless/static@sha256:d2b0ec3141031720cf5eedef3493b8e129bc91935a43b50562fbe5429878d96b
 
 ARG oak_functions_loader_path=/oak_functions_loader
-ARG wasm_module_path=/module.wasm
 
 COPY ./loader/target/x86_64-unknown-linux-musl/release/oak_functions_loader ${oak_functions_loader_path}
 
-# TODO(#1934): Separate the base loader image from that of individual examples.
-COPY ./loader/testdata/non-oak-minimal.wasm ${wasm_module_path}
-
 # We cannot use ARG values as part of ENTRPOINT, so we have to repeat them here.
 # https://docs.docker.com/engine/reference/builder/#environment-replacement
 #
 # Normally we would use the $PORT env variable here, but since we do not have a shell available in
 # the Docker image, we hardcode its default value (8080) here.
 # https://cloud.google.com/run/docs/reference/container-contract#port
 ENTRYPOINT ["/oak_functions_loader", "--http-listen-port=8080", "--wasm-path=/module.wasm"]
+
+# Individual applications must copy their own Wasm module to /module.wasm.
```

Comment:
I think some of these comments could go to a README file, but maybe later.

Reply:
I agree about the README, though this particular comment probably belongs here in the image itself even after we have the README.